src/components/form/form.types.ts

```diff
@@ -20,6 +20,11 @@ export interface FormError {
      */
     name: string;
 
+    /**
+     * Type of the current input field
+     */
+    type?: string;
```

Comment:
I don't think this is correct. The `error.params` field can contain pretty much anything it seems and depends on what the value of `keyword` is according to the docs https://ajv.js.org/api.html#error-parameters

It probably make more sense to expose `params`, but without making any claims on what it is e.g. `Record<string, any>`